Mage.Sets/src/mage/sets/TarkirDragonstorm.java

```diff
@@ -25,6 +25,8 @@ private TarkirDragonstorm() {
         cards.add(new SetCardInfo("Mox Jasper", 246, Rarity.MYTHIC, mage.cards.m.MoxJasper.class));
         cards.add(new SetCardInfo("Narset, Jeskai Waymaster", 209, Rarity.RARE, mage.cards.n.NarsetJeskaiWaymaster.class));
         cards.add(new SetCardInfo("Rally the Monastery", 19, Rarity.UNCOMMON, mage.cards.r.RallyTheMonastery.class));
+        cards.add(new SetCardInfo("Roiling Dragonstorm", 55, Rarity.UNCOMMON, mage.cards.r.RoilingDragonstorm.class));
+        cards.add(new SetCardInfo("Roiling Dragonstorm", 296, Rarity.UNCOMMON, mage.cards.r.RoilingDragonstorm.class));
```

Comment:
if you're adding multiple of the same card to the set you need to use `NON_FULL_USE_VARIOUS` but you can also just put the one version and the others will be added later